tools/generate_report.rb

```diff
@@ -0,0 +1,35 @@
+include ActionDispatch::Routing
```

Comment:
@lpichler Does it make sense to make this run with pure ruby instead of rails runner:

```
#!/usr/bin/env ruby
require File.expand_path('../config/environment', __dir__)  # loads rails
```

If you `chmod +x` this file, you can then invoke it `./tools/generate_report.rb` and if nothing else cares about the pwd, you can run that from any working directory.  What do you think?